Enhancement/5.7/cin 05907 (#72)
cinchy-co/Documentation: release-notes/release-notes/5.7-release-notes.md

  }
}
```
#### Faster query performance for PostgreSQL multi-select column joins

> We optimized query speeds for PostgreSQL users, specifically when joining a multi-select column to its parent table.

Author: ```suggestion
We optimized PostgreSQL queries to be more performant when referencing multi-select columns.
```
